Vic2ToHoI4/Source/HOI4World/Diplomacy/HoI4AIStrategy.h

```diff
@@ -0,0 +1,41 @@
+#ifndef HoI4AISTRATEGY_H_
+#define HoI4AISTRATEGY_H_
+
+
+
+#include "../../V2World/Ai/AIStrategy.h"
+
+
+
+namespace HoI4
+{
+
+class AIStrategy
+{
+  public:
+	explicit AIStrategy(std::string strategyType): type(std::move(strategyType)) {}
```

Comment:
Take the id and value in this constructor so you don't immediately need to call the setter functions.